src/app/space/settings/areas/create-area-dialog/create-area-dialog.component.ts

```diff
@@ -92,7 +86,9 @@ export class CreateAreaDialogComponent implements OnInit {
     if (error.errors.length) {
       error.errors.forEach(error => {
         if (error.status === '409') {
-          this.errors.uniqueValidationFailure = true;
+          this.errors = {
```

Comment:
Please add a call to the global error handler so that this is picked up by Sentry.